tasks/query-cmr/app/query.ts

```diff
@@ -103,3 +141,21 @@ export async function queryGranules(
 
   return catalogs;
 }
+
+/**
+ * Queries a single page (up to 2,000 granules) for the given CMR scrollId,
+ * producing a STAC catalog per granule.  Returns an array of STAC catalogs.
```

Comment:
One catalog per granule? So catalogs are 1:1 with items?

Reply:
Yes. James or Chris can provide more color on that decision if you need it.